ksql-engine/src/main/java/io/confluent/ksql/util/KafkaConsumerGroupClientImpl.java

```diff
@@ -49,9 +57,22 @@ public KafkaConsumerGroupClientImpl(KsqlConfig ksqlConfig) {
 
   @Override
   public List<String> listGroups() {
+    File tmpConfigFile;
+    try {
+      // The ConsumerGroupCommand we use instantiates its own admin client. However, the configs
+      // for the underlying admin client can be passed only through a properties file. So we dump
+      // the admin client configs to a temporary file and then use that file to configure the
+      // underlying admin client correctly.
+      tmpConfigFile = flushPropertiesToTempFile();
+    } catch (IOException e) {
+      log.error("Could not list consumer groups", e);
+      return Collections.emptyList();
```

Comment:
Do we really want to swallow this exception? The `listGroups` can potentially throw, and that isn't being swallowed. It'll be a poor user experience if `SHOW TOPICS` returns an empty list to the user.

How about wrapping entire function in try/catch throwing an appropriate exception up the stack?